luapgsql.c

```diff
@@ -32,7 +32,11 @@
 #define htobe64(x) OSSwapHostToBigInt64(x)
 #elif __linux__
 #include <endian.h>
+#elif _WIN32
+#include <x86intrin.h>
+#define htobe64(x) _bswap64(x)
```

Comment:
Use `htonll` instead. https://msdn.microsoft.com/en-us/library/windows/desktop/jj710199(v=vs.85).aspx
Or check `#if BYTE_ORDER == LITTLE_ENDIAN` and use `__builtin_bswap64`